src/addAndRemove.js

```diff
@@ -0,0 +1,56 @@
+export const addToDo = (input, todoListData) => {
+  const dataObj = {
+    index: todoListData.length,
+    description: '',
+    completed: false,
+  };
+
+  dataObj.description = input;
+  todoListData.push(dataObj);
+};
+
+export const modifyList = (todoListData, saveToLocalStorage, createIndexes, refreshPage) => {
```

Comment:
- [ ] **The `clear all complete` functionality should only clear the completed tasks. Please fix this.**

Currently, if I click the clear button, it clears all the tasks regardless of their completion status. Please fix this and make sure that only the tasks that are completed are removed.

Reply:
Hi @smcommits,
I followed the 'minimalist' Todo List example for the clear buttons. On the minimalist website when you click the clear button on the top left corner, it "clears all tasks". When you click the bottom "Clear all completed" button it only clears only completed tasks. My To-Do List follows the same functionality.

Please clarify if you would want me to apply the same functionality on both buttons; which would mean that both buttons will clear only "completed tasks" and I will not have any button to "clear all tasks" in case one wants to start a completely new to do list.

Reply:
@KDlamini  You are right.  I made a mistake. I am sorry about that. Please request another review after refactoring the function or you can request a second opinion so that your review count is not affected. I will leave a comment on the PR. Again, I am so sorry about the mistake.

Reply:
@smcommits Alright no problem, it happens to all of us. I will continue with refactoring and request a second review. Thanks again for the great review. :)

Reply:
This issue is well resolved! ✔️